src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -590,6 +591,11 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
                 BindingInfo bindingInfo = binding.ToBindingInfo();
 
                 request.Metadata.Bindings.Add(binding.Name, bindingInfo);
+
+                if (binding.SupportsDeferredBinding() && !binding.SkipDeferredBinding())
+                {
+                    _metricsLogger.LogEvent(string.Format(MetricEventNames.BindToParameterBindingData, metadata.Name));
+                }
```

Comment:
@mathewc let me know if we need any log line here giving more information such as - binding name/type and values of `supportsDeferredBinding` and `SkipDeferredBinding`.

Reply:
It might help to have a trace/debug log here

Reply:
For `SkipDeferredBinding`, can you add a trace log [here](https://github.com/Azure/azure-functions-host/blob/afafa87da101d8ed2fe1fa7aee317692db252786/src/WebJobs.Script/Description/FunctionDescriptorProvider.cs#L150) to indicate that we are changing `supportsDeferredBinding` to false because the `SkipDeferredBinding` flag was set

Reply:
It might also be worth a trace log [here](https://github.com/Azure/azure-functions-host/blob/afafa87da101d8ed2fe1fa7aee317692db252786/src/WebJobs.Script/Description/Workers/WorkerFunctionDescriptorProvider.cs#L74) to let us know that an input or output binding contains an expression

Reply:
We have few questions to discuss on logging part. I will start discussion and another PR for logging. We can keep this PR for metrics part.